pkg/kubelet/cadvisor/cadvisor_windows.go

```diff
@@ -31,7 +31,7 @@ type cadvisorClient struct {
 
 var _ Interface = new(cadvisorClient)
 
-// New creates a cAdvisor and exports its API on the specified port if port > 0.
```

Comment:
It doesn't look wrong to me ...

Reply:
Thanks, sorry for setting mistakes PR header.I think it unnecessary, it true?